spec/indexers/hyrax/generic_work_indexer_spec.rb

```diff
@@ -34,7 +34,7 @@
 
     it 'indexes member work and file_set ids' do
       expect(solr_document['member_ids_ssim']).to eq work.member_ids
-      expect(solr_document['generic_type_sim']).to eq ['Work']
+      expect(solr_document['generic_type_si']).to eq 'Work'
```

Comment:
Since both `generic_type_sim` and `generic_type_si` fields are being indexed now, how about including the checks for both?